src/components/formItem/formItem.svelte

```diff
@@ -297,6 +298,10 @@
     display: flex;
     flex-direction: row;
     gap: var(--leo-spacing-s);
+
+    &:empty {
```

Comment:
did you test this works okay in the webcomponent?

Reply:
No, but I will. Pretty sure we've tested this pattern in other cases, though.

Reply:
Confirmed.